media/webview.css

```diff
@@ -6,7 +6,6 @@ FIELDSET, FORM,
 FRAME, FRAMESET,
 H1, H2, H3, H4,
 H5, H6, IFRAME,
-NOFRAMES,
```

Comment:
https://sonarcloud.io/project/issues?id=vscode-didact&issues=AXR4WpWr5CyWyJNY-AoB&open=AXR4WpWr5CyWyJNY-AoB

Reply:
Will deal with this in a separate PR after https://github.com/redhat-developer/vscode-didact/pull/291